server/data/vocabularies.json

```diff
@@ -5,12 +5,11 @@
         "type": "manageable",
         "selection_type": "multi selection",
         "init_version": 2,
```

Comment:
you can bump the init version to force reload

Reply:
I did not change this CV values, just formatting.

Reply:
sorry, my bad